src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +56,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+    @Override
+    public String getPrintableString() {
```

Comment:
Correct implementation of the interface. 
As discussed in tutorial, you may want to consider the privacy setting when returning a printable string in this system.